src/map/npc-parse.cpp

```diff
@@ -427,6 +428,93 @@ bool npc_load_script_function(ast::script::ScriptBody& body, ast::npc::ScriptFun
 }
 
 static
+bool npc_load_script_spell(ast::script::ScriptBody& body, ast::npc::ScriptSpell& script_spell)
+{
+    std::unique_ptr<const ScriptBuffer> script = compile_script(STRPRINTF("script npc \"%s\""_fmt, script_spell.name.data), body, false);
+    if (script == nullptr)
+        return false;
+
+    dumb_ptr<npc_data_script> nd;
+    nd.new_();
+    nd->scr.event_needs_map = false;
+
+    nd->name = script_spell.name.data;
+
+    nd->bl_prev = nd->bl_next = nullptr;
+    nd->bl_m = borrow(undefined_gat);
+    nd->bl_x = 0;
+    nd->bl_y = 0;
+    nd->bl_id = npc_get_new_npc_id();
+    nd->dir = DIR::S;
+    nd->flag = 0;
+    nd->npc_class = INVISIBLE_CLASS;
```

Comment:
this might be problematic for spells that use effects like kaflosh
you might need to put ID 127